### PR TITLE
[SDPA-4658] respect dnt

### PR DESCRIPTION
**Pull request by tim-yao**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-4658

### Changed

1.  Replaced tide gtm module by nuxt gtm module to fix the issue, also removing our duplicated work Nuxt community already has done

### CODE REVIEW

File: packages/ripple-nuxt-tide/lib/mixins/data-layer.js [hunk 1]
```diff
@@ -3,10 +3,8 @@ const dataLayer = {
     if (!this.$gtm) return
     if (!this.page) return
 
-    if (this.page.field_department_agency && this.page.field_department_agency.name) {
-      this.$gtm.pushObj({
-        'department': this.page.field_department_agency.name
-      })
+    if (this.page.field_department_agency?.name) {
```
Comment (rubyllamadora) [inline packages/ripple-nuxt-tide/lib/mixins/data-layer.js:6]: 🥳

File: packages/create-ripple-app/generators/updates/1.31.0/saofile.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+          const defaultNode = rootData.find(j.ExportDefaultDeclaration)
+          defaultNode.find(j.Property, { key: { name: 'gtm' } }).remove()
+          defaultNode.find(j.Property, { key: { name: 'build' } })
+            .insertAfter(`gtm: {
+  id: process.env.GTM_ID,
+  pageTracking: true,
+  pageViewEventName: 'routeChange',
+  noscript: false
+}`)
```
Comment (rubyllamadora) [inline packages/create-ripple-app/generators/updates/1.31.0/saofile.js:33]: 👍

File: examples/vic-gov-au/nuxt.config.js [hunk 3]
```diff
@@ -143,12 +153,7 @@ export default {
       webform: 1,
       search: 1,
       authenticatedContent: 1,
-      alert: 1,
-      gtm: 1
```
Comment (rubyllamadora) [inline examples/vic-gov-au/nuxt.config.js:147]: Did we ever switch this off before for specific site?

**Review by rubyllamadora — APPROVED** [review]

File: examples/vic-gov-au/nuxt.config.js (hunk 3 above)
Comment (tim-yao) [reply examples/vic-gov-au/nuxt.config.js:147]: I think there are two sites have turned it off, but it should be ok. Let them manually update this.

**Pull request merged**